CHANGELOG.md

```diff
@@ -61,6 +61,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- `opentelemetry-instrumentation-flask` Add `http.route` to metric attributes
```

Comment:
You should move this to the next version

Reply:
Done! moved to unreleased section

Reply:
Please move it to the Added section of the unreleased chapter